src/bank01_player.asm

```diff
@@ -696,54 +696,62 @@ scrollRoom:
     ld   A, D
     ld   [wSpriteScrollSpeed], A
 
-    ; Set B to be the 'flip' of the direction in E
-    ld   B, $01
-    bit  1, E ; east
+    ; Set B to be the flip of the direction in E
+    ld   B, DIRECTIONF_EAST
+    bit  DIRECTIONB_WEST, E
     jr   NZ, .check_room_override
-    sla  B
-    bit  0, E ; west
+    sla  B ; B holds DIRECTIONF_WEST
+    bit  DIRECTIONB_EAST, E
     jr   NZ, .check_room_override
-    sla  B
-    bit  3, E ; north
+    sla  B ; B holds DIRECTIONF_NORTH
+    bit  DIRECTIONB_SOUTH, E
     jr   NZ, .check_room_override
-    sla  B
+    sla  B ; B holds DIRECTIONF_SOUTH
```

Comment:
The function `objectReverseDirection` already does this.

Pros of using it:
* Save approximately 15 bytes of space.
* Easier to follow.

Cons:
* Function call overhead.
* Less optimized. (Could be optimized, but currently it handles diagonal directions which isn't bad per se.)
* The name refers to objects.

I'm fine not using it. Honestly, if the name was more generic I'd probably err towards using it, but with it named as it is I think not using it is the correct decision.

Reply:
What about something like this?

```ld a, e
and DIRECTIONF_EAST | DIRECTIONF_WEST
jr z, .north_south
xor DIRECTIONF_EAST | DIRECTIONF_WEST
jr .check_room_override
.north_south:
ld a, e
xor DIRECTIONF_NORTH | DIRECTIONF_SOUTH
ld b, a
```

Untested but the intention is to be functionally equivalent to what you have (aside from cases where both east *and* west or north *and* south are set).

I think it would be faster 3/4 of the time

Reply:
I don't think this section needs optimization. While I agree that's faster, it's not easier to follow. 

I'm ok with using the objectReverseDirection function though. I prefer small code count in bank01. I can rename the function if we don't mind the bloat of changes across bank00, bank01, and bank03.